### PR TITLE
Micro versions bump

### PR DESCRIPTION
**Pull request by rsvoboda**
Micro versions bump for
 * opentracing-tracerresolver
 * smallrye-metrics
 * smallrye-fault-tolerance

### CODE REVIEW

**Review by gsmet — CHANGES_REQUESTED** [review]
I think a rebase is needed as some of them were upgraded this morning.

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -33,10 +33,10 @@
         <microprofile-rest-client.version>1.3.4</microprofile-rest-client.version>
         <smallrye-config.version>1.5.0</smallrye-config.version>
         <smallrye-health.version>2.1.0</smallrye-health.version>
-        <smallrye-metrics.version>2.3.1</smallrye-metrics.version>
+        <smallrye-metrics.version>2.3.2</smallrye-metrics.version>
         <smallrye-open-api.version>1.1.20</smallrye-open-api.version>
         <smallrye-opentracing.version>1.3.2</smallrye-opentracing.version>
-        <smallrye-fault-tolerance.version>2.1.2</smallrye-fault-tolerance.version>
+        <smallrye-fault-tolerance.version>2.1.3</smallrye-fault-tolerance.version>
```
[the remark this answers is not in the document]
Comment (kenfinnigan) [reply bom/runtime/pom.xml]: @Ladicek is probably best placed on this one

File: test-framework/vault/pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <name>Quarkus - Vault - Test</name>
 
     <properties>
-        <test-containers.version>1.12.0</test-containers.version>
+        <test-containers.version>1.12.4</test-containers.version>
```
[the remark this answers is not in the document]
Comment (rsvoboda) [reply test-framework/vault/pom.xml]: It's used here: https://github.com/quarkusio/quarkus/blob/master/test-framework/vault/pom.xml#L34

You mean to remove that dep ?

**Comment by rsvoboda** [comment]
@gsmet rebased
I rebased in the morning and moved to different stuff and forgot how quick quarkus is :)

File: test-framework/vault/pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (gsmet) [reply test-framework/vault/pom.xml]: No, I mean it's now in the BOM so I think we can remove this local version.

It was recently included in the BOM.

File: bom/runtime/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ladicek) [reply bom/runtime/pom.xml]: SmallRye FT 2.1.3 aligns with SmallRye Metrics, so I guess yes, but there's also a bunch of other component updates.

File: test-framework/vault/pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (rsvoboda) [reply test-framework/vault/pom.xml]: I tried to remove that property definition and it failed, I can retry tmr, not at my machine now.

[the remark this answers is not in the document]
Comment (gsmet) [reply test-framework/vault/pom.xml]: And removing the version from the dependency too?

File: bom/runtime/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmartisk) [reply bom/runtime/pom.xml]: These upgrades are specifically required for my PR https://github.com/quarkusio/quarkus/pull/5272/ to work and supersede some things from that PR (if the FT upgrade is merged, I can remove a workaround in my PR), so go ahead, it looks like my PR might take a bit longer to get reviewed and merged.

File: test-framework/vault/pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (rsvoboda) [reply test-framework/vault/pom.xml]:
```diff
diff --git a/test-framework/vault/pom.xml b/test-framework/vault/pom.xml
index 7dc8d1492..f9e87cd7d 100644
--- a/test-framework/vault/pom.xml
+++ b/test-framework/vault/pom.xml
@@ -14,9 +14,9 @@
     <artifactId>quarkus-vault-test</artifactId>
     <name>Quarkus - Vault - Test</name>

-    <properties>
-        <test-containers.version>1.12.4</test-containers.version>
-    </properties>
+<!--    <properties>-->
+<!--        <test-containers.version>1.12.4</test-containers.version>-->
+<!--    </properties>-->

     <dependencies>

@@ -31,7 +31,7 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>postgresql</artifactId>
-            <version>${test-containers.version}</version>
+            <!-- <version>${test-containers.version}</version> -->
             <exclusions>
                 <exclusion>
                     <groupId>javax.xml.bind</groupId>
```

gives 

```
$ mvn clean install
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.testcontainers:postgresql:jar is missing. @ line 31, column 21
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project io.quarkus:quarkus-vault-test:999-SNAPSHOT (/Users/rsvoboda/git/quarkus/test-framework/vault/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.testcontainers:postgresql:jar is missing. @ line 31, column 21
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

[the remark this answers is not in the document]
Comment (rsvoboda) [reply test-framework/vault/pom.xml]: BOM defines dependency `org.testcontainers:testcontainers`, but that module uses `org.testcontainers:postgresql`
Trying `org.testcontainers:testcontainers` in test-framework/vault/pom.xml

[the remark this answers is not in the document]
Comment (rsvoboda) [reply test-framework/vault/pom.xml]: PostgreSQLContainer / POSTGRESQL_PORT are used in vault, so org.testcontainers:postgresql is really needed

[the remark this answers is not in the document]
Comment (gsmet) [reply test-framework/vault/pom.xml]: Just add it to the BOM with the other then. Better that than having 2 versions wandering around.

[the remark this answers is not in the document]
Comment (rsvoboda) [reply test-framework/vault/pom.xml]: PR updated

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**